tests/components/test_nlp.py

```diff
@@ -617,7 +617,7 @@ def test_sys_entity_feature(kwik_e_mart_nlp):
 
 
 test_data_dyn = [
-    ("kadubeesanahalli", None, "store_info", "", ""),
+    ("kadubeesanahalli", None, "store_info", "get_store_hours", ""),
```

Comment:
This test change seems unrelated to the code change.

Reply:
It's in reference to a failing test for all PRs (refer to Minh's comment above). Should I move it to a separate PR?